### DISCUSSION TITLE
changing font style of about section profile

### DISCUSSION THREAD

**Discussion by alieldeba** [community/community#21242] [Other Feature Feedback, Questions, & Ideas]
I am wondering how that guy changes his about section font style I searched a lot and I didn't get anything
I tried to copy it and paste it works!
but when I tried to copy a font from google fonts and paste it, it didn't work

![font](https://user-images.githubusercontent.com/88302640/180749523-8fc958f9-ce66-452b-96ff-64288f8c68a7.png)

Does any help, please?

**Comment by shaido987** [comment]
That is not text with a specific font but unicode symbols. You could try using some text to unicode symbol converter to get the text with the style that you want. For example: https://qaz.wtf/u/convert.cgi?text=Python+Backend+Developer.
